opentelemetry-api/src/opentelemetry/util/_entry_points.py

```diff
@@ -0,0 +1,24 @@
+# Copyright The OpenTelemetry Authors
```

Comment:
since this module already exists and the API of the `entry_points` function is dependent on the python version wouldn't it make sense to do the version checking in here. e.g. this module could provide a generator function with the py 3.10 API to iterate over the entry points and do the version checking internally. the function could be made "private" to avoid exposing it when imported in other modules.

Reply:
Right, fixed.